### PR TITLE
Workspace react header

### PR DESCRIPTION
**Pull request by Gustav-Eikaas**
# Description

- Added icons to header for grid and table
- Added proper styling to status bar items
- Added highlighting of selected tab in header
- Fixed onclick not opening sidesheet
- Fixed workspace body width property causing it to draw out of bounds

Completes: AB#FILL_IN_YOUR_ISSUE_ID

## Type of change

Remove options that are not relevant.

-   [x] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Checklist

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code.
-   [ ] I have linked my DevOps task using the AB# tag.
-   [x] My code is easy to read.
-   [x] Documentation and comments is added where needed.
-   [ ] My code is covered by tests.

### CODE REVIEW

File: packages/workspace-react/src/lib/components/statusBar/StatusBar.tsx [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
 interface StatusBarProps {
 	items: StatusItem[];
 }
 
-export function StatusBar({ items }: StatusBarProps): JSX.Element | null {
+export function StatusBar({ items }: StatusBarProps): JSX.Element {
 	if (!items.length) {
-		return null;
+		//Returns div for styling purposes
+		return <div></div>;
```
Comment (jaysencpp) [inline packages/workspace-react/src/lib/components/statusBar/StatusBar.tsx:12]: i think we can improve this

File: packages/workspace-react/src/lib/components/statusBar/StatusBar.tsx [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
 
 	return (
-		<div style={{ display: 'flex', gap: '2em' }}>
-			{items?.map((s) => (
-				<div title={s.description} key={s.title}>
-					<div>{s.title}</div>
-					<div>{s.value}</div>
-				</div>
+		<StyledStatusBar>
+			{items?.map((item) => (
```
Comment (jaysencpp) [inline packages/workspace-react/src/lib/components/statusBar/StatusBar.tsx:17]: items should not be undefined or empty here right

File: packages/workspace-react/src/lib/components/statusBar/StatusBar.tsx (hunk 1 above)
Comment (Gustav-Eikaas) [reply packages/workspace-react/src/lib/components/statusBar/StatusBar.tsx:12]: I can wrap the component in a div from the parent of this if that helps

File: packages/workspace-react/src/lib/components/workspaceHeader/workspaceHeader.styles.ts [hunk 3]
```diff
@@ -2,8 +2,10 @@ import { tokens } from '@equinor/eds-tokens';
 import styled from 'styled-components';
 
 export const StyledWorkspaceHeader = styled.div`
-	background-color: ${tokens.colors.ui.background__medium.hex};
+	background-color: white;
```
Comment (jaysencpp) [inline packages/workspace-react/src/lib/components/workspaceHeader/workspaceHeader.styles.ts:5]: we should use tokens for everything.. if they have a white color tho

Comment (Gustav-Eikaas) [reply packages/workspace-react/src/lib/components/workspaceHeader/workspaceHeader.styles.ts:5]: tokens.colors.ui.background__default.hex

File: packages/workspace-react/src/lib/hooks/useActiveTab.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+import { WorkspaceViewController } from '../classes';
+import { Tab } from '../types';
+
+export function useActiveTab<TabNames extends string, TError>(controller: WorkspaceViewController<TabNames, TError>) {
+	const [tab, setTab] = useState<Tab<string> | undefined>(
+		controller.tabs.find((s) => s.name === controller.activeTab)
+	);
+
+	useEffect(() => {
+		const { unsubscribe } = controller.onActiveTabChanged((to) => {
```
Comment (jaysencpp) [inline packages/workspace-react/src/lib/hooks/useActiveTab.ts:11]: fix var name "to"

**Review by jaysencpp — APPROVED** [review]

**Pull request merged**